backend/src/test/java/de/sambam/gamenighttracker/service/PlayerServiceTest.java

```diff
@@ -36,4 +36,22 @@ public void addPlayerTester() {
         assertTrue(playerList.contains(Player.builder().id("003").name("Thomas").build()));
 
     }
+
+    @Test
+    @DisplayName("listAllPlayers should return all players")
+    public void listAllPlayersTest() {
+        //GIVEN
+        PlayerDb playerDb = new PlayerDb();
+        playerDb.add(Player.builder().id("001").name("samBAM").build());
+        playerDb.add(Player.builder().id("002").name("Andrea").build());
+
+        //WHEN
+        List<Player> playerList = playerDb.getPlayers();
```

Comment:
you are only testing your PlayerDB, not your PlayerService!

Reply:
changes added : )